linux/centos/7/foxpass_setup.py

```diff
@@ -77,9 +77,9 @@ def write_foxpass_ssh_keys_script(api_url, apis, api_key):
     for api in apis:
         curls.append(base_curl % api)
 
-    with open('/usr/local/bin/foxpass_ssh_keys.sh', "w") as w:
+    with open('/usr/sbin/foxpass_ssh_keys.sh', "w") as w:
```

Comment:
if you want to move this script to sbin, you need to update the reference to it in augment_sshd_config